pkg/datapath/linux/config/config.go

```diff
@@ -390,6 +391,17 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["NATIVE_DEV_MAC_BY_IFINDEX(IFINDEX)"] = macByIfIndexMacro
 		cDefinesMap["IS_L3_DEV(ifindex)"] = isL3DevMacro
 	}
+
+	// Enabling NetfilterCompatibleMode increases BPF instruction memory size, and may
+	// cause issues in kernels have lower memory. Hence enabling, this feature
+	// only in kernels supports higher instruction size.
+	// NetfilterCompatibleMode feature can be enabled on all kernels, once we have support
+	// for adding new tail calls in host device[bpf_host].
```

Comment:
Do you plan to work on this? Do we have a GH issue for it?

Reply:
Sure, I'll work on that. Created GH [issue](https://github.com/cilium/cilium/issues/16952) for tracking purposes.